@ecomplus/storefront-template/template/scss/themes/beauty/_custom.scss

```diff
@@ -0,0 +1,262 @@
+@import "variables";
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+main {
+  .h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6 {
+    color: $secondary;
+  }
+  
+  .h1, .h2, .h3, .h5, h1, h2, h3, h5 {
+    font-family: 'Playfair Display', serif;
+    font-weight: $font-headings;
+  }
+  
+  .h4, .h6, h4, h6 {
+    font-family: 'Poppins', sans-serif;
+    font-weight: $font-headings;
+  }
+
+  .top-bar,
+  .header {
+    background-color: $primary-yiq;
+  }
+
+  .header {
+    border-bottom: 1px solid $gray-200;
+
+    &__nav--full {
+      font-size: $font-size-lg;
+
+      a {
+        padding: 0 $spacer-4;
+      }
+    }
+    &__buttons {
+      .badge-primary {
+        background-color: $purple-light;
+        border-radius: 50%;
+        margin-top: 10px;
+        margin-right: 3px;
+      }
+    }
+  }
+
+  .btn-lg {
+    font-size: 1.5rem;
+  }
+
+  .banner-slider {
+    .glide {  
+      &__bullet,
+      &__bullet--active {
+        background-color: $white;
+        border-color: $gray-900;
+      }
+    }
+    .glide__arrow {
+      display: none;
+    }  
+  }
+  
+  .info-bar {
+    background-color: $primary-yiq;
+    border-bottom: solid 2px $gray-200;
+
+    &__item {
+      color: $secondary;
+      font-family: 'Playfair Display';
+      text-transform: capitalize;
+    
+      &:hover {
+        color: $secondary-text;
+      }
+    }
+  }
```

Comment:
> A info bar precisaria de outro span no html. Como só tem um uma div pro texto em cada item, não tem como colocar as duas font-family, como no xd.

Na verdade mesmo considerando só o "título" em cada coluna na info bar, do protótipo do XD pra cá tem algumas diferenças:
- Cor da fonte;
- Tamanho da fonte;
- Itálico;
- Espessura da fonte;

![Screenshot 2022-04-26 at 01-03-13 Templates E-Com Plus](https://user-images.githubusercontent.com/10326572/165218890-15ff548a-0e13-4f57-bdf0-4259f3a5b7e3.png)